genome_kit/gk_data.py

```diff
@@ -39,10 +39,7 @@
 
 
 eps = entry_points(group="genomekit.plugins.data_manager")
-try:
-    DataManagerImpl = list(eps)[0].load()
-except:
-    DataManagerImpl = DefaultDataManager
+DataManagerImpl = DefaultDataManager
```

Comment:
Won't this change break code that relies on implicitly loading the data manager from a plugin?

Reply:
sorry this was test code